src/amo/pages/Addon/index.js

```diff
@@ -467,6 +465,30 @@ export class AddonBase extends React.Component {
     }
   }
 
+  renderMetaOpenGraph() {
+    const { addon, lang } = this.props;
+
+    const tags = [
+      <meta key="og:type" property="og:type" content="website" />,
+      <meta key="og:url" property="og:url" content={addon.url} />,
+      <meta key="og:title" property="og:title" content={this.getPageTitle()} />,
+      <meta
+        key="og:description"
+        property="og:description"
+        content={this.getPageDescription()}
+      />,
+      <meta key="og:locale" property="og:locale" content={lang} />,
+    ];
+
+    const image = getPreviewImage(addon);
+
+    if (image) {
+      tags.push(<meta key="og:image" property="og:image" content={image} />);
```

Comment:
should we add an alt tag for this too : `og:image:alt`

Reply:
I dont think so because the only content we could add here is the legend of the screenshot but this is not always translated..

Reply:
couldn't we just do 'Image of [addon.name]]`? I see we are doing similar for themes on search etc.

Reply:
I am not sure this is relevant for OG tags to be honest. Consumers of the OG tags will likely use the title because the image should be an illustration of the "node" (and the "node"'s title should describe it).

Reply:
I guess I thought some info was better than nothing but I understand what you're saying..